src/clients/base.rs

```diff
@@ -901,7 +906,7 @@ where
     /// [Reference](https://developer.spotify.com/documentation/web-api/reference/#endpoint-get-recommendations)
     async fn recommendations<'a>(
         &self,
-        payload: &Map<String, Value>,
+        attributes: impl IntoIterator<Item = (&'a str, Value)> + Send + 'a,
```

Comment:
Is it better to make attribute as a named type(such as `Enum`) instead of a string?

Reply:
Done. I wanted to avoid the huge enum but it's indeed much easier to use this way.